src/routes/FAQ.svelte

```diff
@@ -1,17 +1,18 @@
 <script>
+  import { TranslatedStrings } from "$lib/translations/translations";
   import { Alert, Button } from "flowbite-svelte";
+  import discord from "$assets/images/Discord.png";
 </script>
 
 <div class="ml-20">
   <div class="flex flex-col h-5/6 p-8">
     <Alert color="dark" rounded={false} accent>
       <span class="text-lg font-medium text-black-900 dark:text-blue-800"
-        >Help & Support</span
+        >{TranslatedStrings.faq_help}</span
```

Comment:
I mentioned that you should destructure the `TranslatedStrings` object. It's nice that you're doing it in some places, but you should do it everywhere.

Reply:
Oh i forgot that, gonna do that next